### PR TITLE
Us29 admin merchant create

### PR DESCRIPTION
**Pull request by abwmn**

### CODE REVIEW

File: app/views/admin/merchants/_merchant_partial.html.erb [hunk 1]
```diff
@@ -1,4 +1,6 @@
 <div class="merchant<%= merchant.id %>">
-  <%= link_to merchant.name, admin_merchant_path(merchant) %><br>
-  <%= button_to (merchant.enabled ? 'Disable Merchant' : 'Enable Merchant'), admin_merchant_path(merchant, switch_enabled: true), method: :patch %>
+  <p><%= link_to merchant.name, admin_merchant_path(merchant) %> 
+  <%= button_to (merchant.enabled ? "Disable #{merchant.name}" : "Enable #{merchant.name}"),
```
Comment (amspears007) [inline app/views/admin/merchants/_merchant_partial.html.erb:3]: Nice use of ternary operator!!!! `?`

**Review by amspears007 — APPROVED** [review]

File: config/routes.rb [hunk 2]
```diff
@@ -13,6 +13,6 @@
 
   namespace :admin do
     resources :invoices, only: [:show]
-    resources :merchants, only: [:index, :show, :edit, :update], controller: 'merchants'
+    resources :merchants, except: [:destroy], controller: 'merchants'
```
Comment (sicknacco) [inline config/routes.rb:16]: Thank you for doing this refactor

File: spec/features/admin/merchants/index_spec.rb [hunk 3]
```diff
@@ -27,25 +27,45 @@
...
+
+      it 'I see a link to create a new merchant with a default status of disabled' do
+        visit admin_merchants_path
+        
+        expect(page).not_to have_content('Marchand')
+  
+        click_link 'Create New Merchant'
+        expect(current_path).to eq(new_admin_merchant_path)
+  
+        fill_in 'Name', with: 'Marchand'
```
Comment (thawley2) [inline spec/features/admin/merchants/index_spec.rb:58]: One of my favorite merchants :)

**Review by sicknacco — APPROVED** [review]
Looks great! Thank you

**Review by thawley2 — APPROVED** [review]
Looks good you two!!! killin it!!!

**Pull request merged**